packages/devtools_testing/lib/support/http_request_timeline_test_data.json

```diff
@@ -1 +1,5764 @@
Dart","tid":52414,"pid":52406,"ts":231936857550,"ph":"b","id":"41","args":{"filterKey":"HTTP/client","method":"HEAD","uri":"http://127.0.0.1:8011/foo/bar","isolateId":"isolates/3907117677703047"}},{"name":"HTTP CLIENT HEAD","cat":"Dart","tid":52414,"pid":52406,"ts":231936858577,"ph":"e","id":"41","args":{"error":"SocketException: OS Error: Connection refused, errno = 111, address = 127.0.0.1, port = 35296","isolateId":"isolates/3907117677703047"}},{"name":"HTTP CLIENT POST","cat":"Dart","tid":52414,"pid":52406,"ts":231936870576,"ph":"b","id":"42","args":{"filterKey":"HTTP/client","method":"POST","uri":"http://127.0.0.1:8011/foo/bar?foo=bar&year=2019","isolateId":"isolates/3907117677703047"}},{"name":"HTTP CLIENT POST","cat":"Dart","tid":52414,"pid":52406,"ts":231936871628,"ph":"e","id":"42","args":{"error":"SocketException: OS Error: Connection refused, errno = 111, address = 127.0.0.1, port = 35298","isolateId":"isolates/3907117677703047"}},{"name":"HTTP CLIENT CONNECT","cat":"Dart","tid":52414,"pid":52406,"ts":231936907550,"ph":"b","id":"43","args":{"filterKey":"HTTP/client","method":"CONNECT","uri":"http://127.0.0.1:8011/foo","isolateId":"isolates/3907117677703047"}},{"name":"HTTP CLIENT CONNECT","cat":"Dart","tid":52414,"pid":52406,"ts":231936908597,"ph":"e","id":"43","args":{"error":"SocketException: OS Error: Connection refused, errno = 111, address = 127.0.0.1, port = 35300","isolateId":"isolates/3907117677703047"}},{"name":"HTTP CLIENT PATCH","cat":"Dart","tid":52414,"pid":52406,"ts":231936931591,"ph":"b","id":"44","args":{"filterKey":"HTTP/client","method":"PATCH","uri":"http://127.0.0.1:8011/foo/bar/baz?foo=bar&year=2019","isolateId":"isolates/3907117677703047"}},{"name":"HTTP CLIENT PATCH","cat":"Dart","tid":52414,"pid":52406,"ts":231936932681,"ph":"e","id":"44","args":{"error":"SocketException: OS Error: Connection refused, errno = 111, address = 127.0.0.1, port = 35302","isolateId":"isolates/3907117677703047"}},{"name":"HTTP CLIENT POST","cat":"Dart","tid":52414,"pid":52406,"ts":231937016599,"ph":"b","id":"45","args":{"filterKey":"HTTP/client","method":"POST","uri":"http://127.0.0.1:8011?foo=bar&year=2019","isolateId":"isolates/3907117677703047"}},{"name":"HTTP CLIENT POST","cat":"Dart","tid":52414,"pid":52406,"ts":231937017768,"ph":"e","id":"45","args":{"error":"SocketException: OS Error: Connection refused, errno = 111, address = 127.0.0.1, port = 35304","isolateId":"isolates/3907117677703047"}},{"name":"HTTP CLIENT POST","cat":"Dart","tid":52414,"pid":52406,"ts":231937063645,"ph":"b","id":"46","args":{"filterKey":"HTTP/client","method":"POST","uri":"http://127.0.0.1:8011/foo/bar","isolateId":"isolates/3907117677703047"}},{"name":"HTTP CLIENT POST","cat":"Dart","tid":52414,"pid":52406,"ts":231937065295,"ph":"e","id":"46","args":{"error":"SocketException: OS Error: Connection refused, errno = 111, address = 127.0.0.1, port = 35306","isolateId":"isolates/3907117677703047"}}],"timeOriginMicros":231932850307,"timeExtentMicros":4214988}
```

Comment:
This file is identical to the previous file except that I intentionally omitted one start event to trigger a problem case I was seeing with a real world app where everything crashed if the first network event was incomplete due to the start of the network event being before the start of the snapshot.